Qrcode branding

arjunsudheer opened the pull request:
Update qr_code.py to use pyqrcode library. Add SCE branding to generated QR codes.

arjunsudheer committed:
Add SCE_logo.png to add branding to generated qr codes

Update Dockerfile to copy SCE_logo.png to the Docker container

arjunsudheer committed:
Update add method in qr_code.py to use pyqrcode library instead of qrcode

Add branding with SCE logo for generated qr codes, update requirements.txt with pyqrcode and pypng dependencies

evanugarte commented on Dockerfile:
```diff
@@ -4,6 +4,8 @@ WORKDIR /app
 
 RUN apt-get update && apt-get install -y default-libmysqlclient-dev
 
+COPY ./assets ./assets
+
```
can we mount this in the docker-compose ymls like
```yml
volumes:
- ./assets:/app/assets
```

then we can remove this line

evanugarte commented on modules/qr_code.py:
```diff
@@ -1,10 +1,12 @@
-import qrcode
+import pyqrcode
 import uuid
 import os
 import logging
+from PIL import Image
 
```
```suggestion
import uuid
import os
import logging

import pyqrcode
from PIL import Image

```

evanugarte reviewed (commented):
looks very good, small comments

arjunsudheer committed:
Update code from code review

Mount assets directory as a volume instead of copying it in the Dockerfile, add separation between import statements in qr_code.py

evanugarte committed:
reorder imports
